Styles.css

```diff
@@ -1,31 +1,358 @@
+    border-bottom: 50px solid white;
+}
 
 /* --------------- 4.2 Introduction ------------------- */
+.introduction {
+    max-width: 700px;
+    margin-top: -2rem;
+}
+
+.introduction h2 {
+    color: var(--purple);
+    text-align: center;
+}
+
+.introduction p {
+    margin-top: 2rem;
+    text-align: center;
+}
+
+.triangle-introduction {
+    width: 0;
+    height: 0;
+    border-right: 100vw solid white;
+    border-bottom: 50px solid var(--projects-background);
+}
+
+/* --------------- 4.3 Projects ------------------- */
+.projects {
+    background-color: var(--projects-background);
+}
+
+.projects article {
+    display: flex;
+    justify-content: space-between;
+    gap: 2rem;
+    align-items: center;
+    margin-top: 4rem;
+    z-index: 2;
+}
+
+.projects article:nth-child(even) {
+    flex-direction: row-reverse;
+}
+
+.projects article div {
+    display: flex;
+    flex: 0 0 300px;
+    flex-direction: column;
+    min-height: 200px;
+    justify-content: space-between;
+}
+
+.projects article div img {
+    width: 50px;
+    margin-top: -2rem;
+}
+
+.projects span {
+    flex-basis: 600px;
+    box-shadow: 0 10px 20px rgb(0, 0, 0, 0.3);
+}
+
+.projects .background-text {
+    color: white;
+    opacity: 0.5;
+    z-index: 1;
+}
+
+.triangle-projects {
+    width: 0;
+    height: 0;
+    border-right: 100vw solid var(--projects-background);
+    border-bottom: 30px solid transparent;
+}
+
+/* --------------- 4.4 Services ------------------- */
+.services {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: center;
+    margin-top: 4rem;
+    position: relative;
+}
+
+.services article {
+    display: flex;
+    flex-direction: column;
+    flex-basis: 300px;
+    height: 250px;
+    padding: 2rem;
+    border: 1px solid var(--purple);
+    box-shadow: 0 10px 10px rgb(0, 0, 0, 0.1);
+    background-color: white;
+}
+
+.services img {
+    width: 50px;
+}
+
+.services h3 {
+    margin-block: 0.5rem;
+    color: var(--purple);
+}
+
+.services p {
+    color: var(--services-text);
+}
+
+.services .background-text {
+    font-size: 14rem;
+    top: 1rem;
+}
+
+/* --------------- 4.5 Contact ------------------- */
+.contact {
+    background: rgb(87, 34, 205);
+    background: linear-gradient(270deg, rgba(87, 34, 205, 1) 0%, rgba(72, 69, 228, 1) 100%);
+    color: white;
+}
+
+.contact form {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    max-width: 700px;
+    height: 500px;
+}
+
+.contact button {
+    width: 120px;
+    align-self: center;
+}
+
+.contact input, textarea {
+    padding: 15px;
+    font-family: inherit;
+    border-radius: 3px;
+    border: none;
+    margin-bottom: 2rem;
+}
+
+.triangle-contact {
+    width: 0;
+    height: 0;
+    border-right: 100vw solid white;
+    border-bottom: 30px solid transparent;
+}
+
+/* --------------- 4.6 Footer ------------------- */
+footer {
+    padding: 1rem;
+    text-align: center;
+    color: white;
```

Comment:
Je maakt keurig gebruik van root variables voor meerdere kleuren, maar waarom dan niet voor deze? De kleur white heb je 10x toegewezen in jouw CSS.